Cubical/Data/Int/Properties.agda

```diff
@@ -101,98 +101,74 @@ discreteInt (negsuc n) (negsuc m) with discreteℕ n m
 isSetInt : isSet Int
 isSetInt = Discrete→isSet discreteInt
 
--- Compose sucPathInt with itself n times. Transporting along this
--- will be addition, transporting with it backwards will be
--- subtraction.
++pos : ℕ → Int → Int
```

Comment:
Maybe make this infix? And also maybe call it `pos+` as the positive number is the first argument.

Reply:
The reason I used `+pos` was 

```
m + pos n = +pos n m
```

so if you think `+pos n` as an operator, it's action on m will be `m + pos n`.

I think infix and swap the order of it's argument will make it work better. So we have

```
m + pos n = m +pos n
```

Reply:
Yeah, that's clearer!